.lgtm.yml

```diff
@@ -8,3 +8,6 @@ extraction:
     index:
       filters:
         - exclude: "**/*.js"
+queries:
+  - exclude: py/missing-equals
+  - exclude: py/import-and-import-from
```

Comment:
maybe with this you can revert some annoying changes eg the "import nibabel as nib" that disappeared and stuff related to os vs. op

I don't like that we don't keep the origin of the function in the call :)

Reply:
:(